docs/storage.md

```diff
@@ -5,19 +5,26 @@ sort_rank: 5
 
 # Storage
 
-Prometheus includes a local on-disk time series database, but also optionally integrates with remote storage systems.
+Prometheus includes a local time series database, but also optionally integrates with remote storage systems.
```

Comment:
on-disk is a key piece of information here, it's not in RAM

Reply:
Nor necessarily on something that spins.  Reverted.